src/CMakeLists.txt

```diff
@@ -21,3 +21,16 @@ endif()
 
 # Install
 install(TARGETS openblok RUNTIME DESTINATION ${EXEDIR})
+
+if(VITA)
+    include("$ENV{VITASDK}/share/vita.cmake" REQUIRED)
+
+    vita_create_self(openblok.self openblok)
+
+    vita_create_vpk(openblok.vpk "OPBL00001" openblok.self
+        VERSION "01.00"
+        NAME OpenBlok
+        FILE ${CMAKE_CURRENT_SOURCE_DIR}/../etc/vita/sce_sys sce_sys
+        FILE ${CMAKE_CURRENT_SOURCE_DIR}/../data data
```

Comment:
There's a CMake variable for the directory of the top level CMake file, but I forgot which one.

Reply:
Oh, using ``CMAKE_SOURCE_DIR`` works